.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes #
+
+#### Checklist
+
+- [ ] I have read the Contribution & Best practices Guide and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added necessary documentation (if appropriate)
+
+#### Changes proposed in this pull request:
+
+-
+-
```

Comment:
would be nice to have a _yes/no_ question, if this PR introduces a breaking change (e.g. some breaking API change on the server or client SDK).

Reply:
I think we can leave this to the contributor.